logger/src/commonMain/kotlin/com/wire/kalium/logger/KaliumLogger.kt

```diff
@@ -0,0 +1,109 @@
+package com.wire.kalium.logger
+
+import co.touchlab.kermit.Severity
+import co.touchlab.kermit.StaticConfig
+import co.touchlab.kermit.Logger as KermitLogger
+
+/**
+ * LoggerType in order from lowest to the highest severity level.
+ */
+enum class LoggerType {
```

Comment:
Isn't this often called LogLevel?

Reply:
Decided to rename it to: `KaliumLogLevel` as `LogLevel` was ambiguous due to Kermit lib.